pyzeebe/channel/utils.py

```diff
@@ -5,9 +5,123 @@
 DEFAULT_ZEEBE_ADDRESS = "localhost:26500"
 
 
-def create_address(
-    grpc_address: str | None = None,
-) -> str:
-    if grpc_address:
-        return grpc_address
-    return os.getenv("ZEEBE_ADDRESS", DEFAULT_ZEEBE_ADDRESS)
+def get_zeebe_address(default_value: str | None = None) -> str:
+    """
+    Args:
+        default_value (str, optional): Default value to be used if no other value was discovered.
+    Returns: Value from ZEEBE_ADDRESS environment variable
+            or provided default_value
+            or "localhost:26500"
+    """
+
+    return os.getenv("ZEEBE_ADDRESS") or default_value or DEFAULT_ZEEBE_ADDRESS
```

Comment:
In the documentation only ever default is mentioned not default_value, also l like default more (shorter, simple).

Also docstring changes to make it compatible with sphinx. Like `Returns` with a type or in the other functions `Raises`.

```suggestion
def get_zeebe_address(default: str = None) -> str:
    """
    Get the Zeebe Gateway Address.

    Args:
        default (str, optional): Default value to be used if no other value was discovered.

    Returns:
        str: ZEEBE_ADDRESS environment variable or provided default or "localhost:26500"
    """
    return os.getenv("ZEEBE_ADDRESS") or default or DEFAULT_ZEEBE_ADDRESS
```